### PR TITLE
Enhance example translation files

### PR DESCRIPTION
**Pull request by rubo77**
This is the first suggestion, for a start in german. if we agree on a version, I will translate the english accordingly.

- added different wordings in the novpn section, than in the pupkey section
- corrected the reset-button time to press
- added some DSGVO conform texts to the email (which was missing in https://github.com/freifunk-gluon/gluon/pull/1394 )
- corrected the comment, so it is obvious, that it is meant for the next two lines

we could also add an example like "Knotenalarm", that probably many communities would like to implement one day, so it is usefull to collect the agreement already beforehand (We use [ff-node-monitor](https://github.com/freifunk-saar/ff-node-monitor) in Kiel, which works fine).

### CODE REVIEW

**Comment by rubo77** [comment]
Should I translate this like it is?

File: docs/site-example/i18n/de.po [hunk 1]
```diff
@@ -28,35 +28,63 @@ msgstr ""
...
+"wurde, kann sich dein Knoten mit dem Mesh-VPN dort verbinden. Bitte schicke "
+"dazu diesen Schlüssel und den Namen deines Knotens "
 "(<em><%=pcdata(hostname)%></em>) an "
-"<a href=\"mailto:keys@alpha-centauri.freifunk.net?subject=<%= urlencode('Anmeldung: ' .. hostname) %>&amp;"
-"body=<%= urlencode('# ' .. hostname .. '\n' .. pubkey) %>\">keys@alpha-centauri.freifunk.net</a>."
+"<a href=\"mailto:keys@alpha-centauri.freifunk.net?subject="
+"<%= urlencode('Anmeldung: ' .. hostname) %>&amp;body="
+"<%= urlencode('# ' .. hostname .. '\n# ' .. sysconfig.primary_mac .. '\nkey ') %>"
+"%22<%= pubkey %>%22;"
+"<%= urlencode('\n\nIch habe zur Kenntnis genommen, dass der im ') %>"
```
Comment (rotanid) [inline docs/site-example/i18n/de.po]: not every community publishes the contact. the only thing which applies to every community i think would be, that the contact can be seen by anyone using the specific Freifunk network (via respondd)

File: docs/site-example/i18n/de.po [hunk 2]
```diff
@@ -28,35 +28,63 @@ msgstr ""
...
+"<%= urlencode('Knoten von mir eingetragene Kontakt im Internet ') %>"
+"<%= urlencode('öffentlich einsehbar ist und von beliebigen Diensten ') %>"
+"<%= urlencode('(z.B. der Freifunk-Karte) verwendet werden kann.') %>"
+"<%= urlencode('\n\nIch möchte informiert werden, falls mein Knoten ') %>"
```
Comment (rotanid) [inline docs/site-example/i18n/de.po]: please don't add stuff that doesn't apply to most communities

File: docs/site-example/i18n/de.po (hunk 1 above)
Comment (rubo77) [reply docs/site-example/i18n/de.po]: I thought, that everyone can ask respondd for the contact in batman anyway, also if the contact is not shown on the map.

File: docs/site-example/i18n/de.po [hunk 3]
```diff
@@ -28,35 +28,63 @@ msgstr ""
...
+"\">keys@alpha-centauri.freifunk.net</a>. Deine E-Mail Adresse wird "
+"selbstverständlich vertraulich behandelt und nicht weitergegeben."
 "</p>"
 "<div class=\"the-key\">"
-" # <%= pcdata(hostname) %>"
-" <br />"
+"# <%= pcdata(hostname) %><br />"
 "<%= pubkey %>"
 "</div>"
+"<p>Dein Knoten startet gerade neu und wird anschließend versuchen, sich mit "
```
[the remark this answers is not in the document]
Comment (rubo77) [reply docs/site-example/i18n/de.po]: Weis sich der Text unterscheidet, je nachdem ob man mesh-vpn aktiv hat oder nicht.

File: docs/site-example/i18n/de.po (hunk 1 above)
Comment (rotanid) [reply docs/site-example/i18n/de.po]: sure, but respondd doesn't answer to public ipv6 requests - and even if it does, not every community has public ipv6

File: docs/site-example/i18n/de.po (hunk 3 above)
[the remark this answers is not in the document]
Comment (rotanid) [reply docs/site-example/i18n/de.po]: ok, got it.

File: docs/site-example/i18n/de.po [hunk 4]
```diff
@@ -28,35 +28,63 @@ msgstr ""
...
+"anderen Freifunkknoten in seiner Nähe und über deine Internetleitung, online "
```
Comment (rotanid) [inline docs/site-example/i18n/de.po]: i think the following is sufficient: "und über deine Internetverbindung mit einem"

File: docs/site-example/i18n/de.po [hunk 5]
```diff
@@ -28,35 +28,63 @@ msgstr ""
...
+"mit einem Freifunk-Server zu verbinden.</p>"
```
Comment (rotanid) [inline docs/site-example/i18n/de.po]: what's a "Freifunk-Server"? please use more appropriate wording like ...
"in seiner Nähe über WLAN sowie über deine Internetverbindung über das VPN-Gateway zu verbinden."
though i'm not sure about "VPN-Gateway"

File: docs/site-example/i18n/de.po (hunk 1 above)
Comment (rubo77) [reply docs/site-example/i18n/de.po]: Ich denke eher, dass der text so gemeint war, dass die contact adresse dort halt von dritten abgefragt werden kann, die auch im Freifunk Netz sind

Comment (rubo77) [reply docs/site-example/i18n/de.po]: ok, i changed it.

**Pull request merged**